src/component/plotly-dash-preview/render.js

```diff
@@ -31,35 +33,53 @@ function render (info, opts, sendToMain) {
    * We check for a 'waitfor' div in the dash-app
    * which indicates that the app has finished rendering.
    */
-  const finishedLoading = () => {
+  const loaded = () => {
     return win.webContents.executeJavaScript(`
       new Promise((resolve, reject) => {
-        var a = document.getElementById("waitfor");
-        if (a) {
-          resolve(true);
+        let tries = ${cst.maxRenderingTries}
+
+        if (${info.timeOut}) {
+          tries = parseInt(${info.timeOut} * 1000/${cst.minInterval})
```

Comment:
Let's move the coercion step to `parse`. Note that `Number(info.timeOut)` would suffice. `parseInt` is slow.

Reply:
:cow2: `1000 / ${cst.minInterval)` please.